man/docker-cp.1.md

```diff
@@ -7,87 +7,88 @@ docker-cp - Copy files/folders between a container and the local filesystem.
 # SYNOPSIS
 **docker cp**
 [**--help**]
-CONTAINER:PATH LOCALPATH|-
+CONTAINER:SRC_PATH DEST_PATH|-
 
 **docker cp**
 [**--help**]
-LOCALPATH|- CONTAINER:PATH
+SRC_PATH|- CONTAINER:DEST_PATH
 
 # DESCRIPTION
 
 In the first synopsis form, the `docker cp` utility copies the contents of
-`PATH` from the filesystem of `CONTAINER` to the `LOCALPATH` (or stream as
+`SRC_PATH` from the filesystem of `CONTAINER` to the `DEST_PATH` (or stream as
```

Comment:
The general description in the man page is different from the `docs/...` page, I think @moxiegirl meant to make these the same

@moxiegirl am I correct that you wanted that?